whaledo/transforms.py

```diff
@@ -231,11 +230,10 @@ def __len__(self) -> int:
     def __add__(self, other: Self) -> Self:
         copy = gcopy(self, deep=False)
         copy.global_views = copy.global_views + other.global_views
-        if copy.local_views is None:
```

Comment:
other is required to be of the same type (Self) so whether copy or other is used for the checks is irrelevant here